src/transformers/models/auto/modeling_auto.py

```diff
@@ -888,6 +904,17 @@ class AutoModelForVisualQuestionAnswering(_BaseAutoModelClass):
 )
 
 
+class AutoModelForDocumentQuestionAnswering(_BaseAutoModelClass):
+    _model_mapping = MODEL_FOR_DOCUMENT_QUESTION_ANSWERING_MAPPING
+
+
+AutoModelForDocumentQuestionAnswering = auto_class_update(
+    AutoModelForDocumentQuestionAnswering,
+    head_doc="document question answering",
+    checkpoint_for_example="impira/layoutlm-document-qa",
```

Comment:
To be updated. Maybe we could fine-tune a LayoutLM model for DocVQA, after which we transfer it to the `huggingface` org?

Reply:
Oh nvm, I just saw that your part of the `impira` organization :D very cool. 

Maybe, with your consent, could we copy this checkpoint to the `huggingface` organization,  to make sure we can maintain it?

Reply:
That makes sense. Maybe we can discuss live quickly what the best way to do this is? We plan to keep improving this model and training it, and I think it'd be good to find a way to maybe collaboratively maintain it?

Reply:
We can ping a revision in that case, i.e. `from_pretrained("impira/layoutlm-document-qa", revision=...")` with ... being a certain commit hash.

Reply:
ok cool, i can do that. Let's wait for PR #18407 in case that results in any more changes, and then I'll pin a revision.

Reply:
We only need a revision and that should be enough! Think it's fine to have community models being used by default

Reply:
Thanks @patrickvonplaten! To clarify my plan then -- I will wait for PR #18407 to land, and then I'll update the `impira/layoutlm-document-qa` model to remove the "remote" pipeline that's currently there, and then update this revision. Let's keep this comment thread open until then.